### ISSUE TITLE
Provide process to schedule notice expiry

### ISSUE THREAD

**Issue by tomkerkhove** [azure-deprecation/infrastructure#8]
Provide a process that schedules notice expiry on a Service Bus queue.

It should:
1. Subscribe for new deprecation announcement events
2. Schedule closure 1 day after the deprecation date

Depends on #36

**Comment by tomkerkhove** [comment]
Duplicate of https://github.com/azure-deprecation/infrastructure/issues/10
